File: public/phase-2-20160620022203/C_SHARP/colored-balls.cs
// Name : Colored Balls
using System;
namespace Application {
	class colored_balls {
		public static void Main(string[] args) {
		// input:
		// k :  the number of colors
		// c :  the number of balls of each color 
		int idx;
		int jdx;
		int k = Convert.ToInt32(Console.ReadLine());
		int[,] c = new int[k,1];
		string[] c_elements = new string[100];
		for(idx=0;idx<k;idx++){
			c_elements = (Console.ReadLine()).Split(' ');
			for(jdx=0;jdx<1;jdx++){
				c[idx,jdx] = Convert.ToInt32(c_elements[jdx]);
			}
		}


		// write your code here
		// store your results in `result`

		// output
		// Dummy Data
		int result=3;

		Console.WriteLine(result);

		}	
	}
}

File: public/phase-2-20160620022203/C/colored-balls.c
// Name : Colored Balls
#include <stdio.h>
int main() {
	// input:
	// k :  the number of colors
	// c :  the number of balls of each color 
	int k, c[1000][1], result=3, idx, jdx;
	char c_tmp[1];
	scanf("%d", &k);
	for(idx = 0; idx< k; idx++) {
		scanf("%s", &c_tmp[0]);
		for(jdx = 0;jdx<1;jdx++) {
			c[idx][jdx] = c_tmp[jdx];
		}
	}


	// write your code here
	// store your results in `result`

	// output
	printf("%d", result);
	return 0;
}